### DISCUSSION TITLE
Changing labels/tags from the issues reported

### DISCUSSION THREAD

**Discussion by renatoyamane** [AdguardTeam/AdGuardHome#6925] [General]
Sorry for my stupid question, but I reported an issue on the link below and it's tagged with "Waiting for data", but I already submitted the data requested.
How can I remove this tag/label? :-)
https://github.com/AdguardTeam/AdGuardHome/issues/6897

**Comment by ghost** [comment]
Hi there, apologies, I've been quite busy lately. I'll get it removed now.

**Reply by renatoyamane** [reply]
I thought I was missing something and I should be the person in charge to change that label...
Thanks!!
